processing/src/main/java/org/apache/carbondata/processing/sortandgroupby/sortdata/SortIntermediateFileMerger.java

```diff
@@ -17,6 +17,7 @@
 package org.apache.carbondata.processing.sortandgroupby.sortdata;
 
 import java.io.File;
+import java.util.Random;
```

Comment:
this will cause checkstyle problems

`import` should be organized in alphabetical order.

You can let it IDE import automatic instead of manually.